README.md

```diff
@@ -1,21 +1,21 @@
 # Craft CLI
 
-A Command Line Client builder that follows the [Canonical's Guidelines for a Command Line
-Interface](https://discourse.ubuntu.com/c/design/cli-guidelines/62).
+A Command Line Client builder that follows the [Canonical's Guidelines for a
+Command Line Interface](https://discourse.ubuntu.com/c/design/cli-guidelines/62).
 
-The library provides two main functionalities: 
+The library provides two main functionalities:
 
-- a framework to define and execute application commands, which involves argument parsing and the provision of help texts
+- a framework to define and execute application commands, which involves
```

Comment:
Do not cut the lines, just let them flow. It's slightly worse when you see it not rendered, but you don't get the "lot of lines rewrapping diff mess" when changing the text.